src/pages/docs/go-w3up.mdx

```diff
@@ -30,10 +30,19 @@ go get github.com/web3.storage/go-ucanto
 
 ## Generate a DID
 
-You can use `ucan-key` to generate a private key and DID for use with the library. Install Node.js and then use the `ucan-key` module:
+Currently the best way to generate a DID is to use the w3up JS CLI:
+
+<Steps>
+### Install w3 CLI
+
+```sh
+npm install -g @web3-storage/w3cli
+```
+
+### Generate a DID
 
 ```sh
-npx ucan-key ed
+w3 key create
```

Comment:
Maybe leave this as `ucan-key` as it has fewer dependencies?

Or for the sake of go devs patience, they will still be able to use npx like `npx @web3-storage/w3cli key create`, as `key create` doesn't need any existing client state.

Reply:
can confirm... npx still works

```shell
$ npx @web3-storage/w3cli key create
# did:key:z6Mkuu8gnJEe5uWPtExfwPcC8xvJQr23AmXArmqKXwBYGCEV
MgCZa59TQQ+xTk2ko+Sn+oZZ3ir9bmD6gZ5s09ubqk9I3bO0B5YAVloPr3OhRk9qSEMgV1sHaQFz6mCTeJeYrJkWxL6I=
```